utils/sql/src/main/resources/io/apicurio/registry/storage/impl/sql/h2.ddl

```diff
@@ -25,7 +25,9 @@ ALTER TABLE content ADD CONSTRAINT UNQ_content_1 UNIQUE (contentHash);
 CREATE INDEX IDX_content_1 ON content(canonicalHash);
 CREATE INDEX IDX_content_2 ON content(contentHash);
 
-CREATE TABLE versions (globalId BIGINT AUTO_INCREMENT NOT NULL, artifactId VARCHAR(512) NOT NULL, version INT NOT NULL, state VARCHAR(64) NOT NULL, name VARCHAR(512), description VARCHAR(1024), createdBy VARCHAR(256), createdOn TIMESTAMP NOT NULL, labels CLOB, properties CLOB, contentId BIGINT NOT NULL);
+CREATE SEQUENCE globalidsequence;
+
+CREATE TABLE versions (globalId BIGINT NOT NULL, artifactId VARCHAR(512) NOT NULL, version INT NOT NULL, state VARCHAR(64) NOT NULL, name VARCHAR(512), description VARCHAR(1024), createdBy VARCHAR(256), createdOn TIMESTAMP WITHOUT TIME ZONE NOT NULL, labels TEXT, properties TEXT, contentId BIGINT NOT NULL);
```

Comment:
If we change the date type to add `WITHOUT TIME ZONE` then I think we should do it in the `artifacts` table too.  Also we should change any other `CLOB` types to `TEXT`.  Should be consistent...

Reply:
I don't remember changing that column type, I'll return that to `TIMESTAMP NOT NULL`

Reply:
No I think the change is good - and is more consistent with PG!  :)  Let's just make the same change to the other types.  I think when I looked up the H2 types I must not have realized they supported that.  Same with `TEXT`.